submissions/xT0PERx/OOP exercise/index.js

```diff
@@ -0,0 +1,93 @@
+class Mammals {
+  constructor(props) {
+    this.name = props.name;
+    this.saying = props.saying;
+  }
+  say() {
+    return this.saying;
+  }
+  printInfo() {
+    print(
```

Comment:
Way too specific use case. What if we want a different method of output? Wouldn't be great to add yet another method to handle just it. Class shouldn't be aware of interaction interfaces.
Well, this method might receive a callback to apply as an output method. The simpler approach is to have a method that simply returns a string and inquiring party does with that string whatever it likes.